attributes/default.rb

```diff
@@ -69,7 +69,8 @@
   default['squid']['package'] = 'squid3'
   default['squid']['config_dir'] = '/etc/squid3'
   default['squid']['config_file'] = '/etc/squid3/squid.conf'
-  default['squid']['service_name'] = 'squid3'
+  default['squid']['service_name'] = 'squid3' if node['platform_version'].to_i <  16
```

Comment:
There is already a default on line 37.

Here my change is in "case-when". I can add an "if" after it if you think it's better but it will decrease readability.